src/agent/src/random.rs

```diff
@@ -13,7 +13,7 @@ use tracing::instrument;
 
 pub const RNGDEV: &str = "/dev/random";
```

Comment:
I'm wondering if we really want to use `/dev/random` as it blocks the caller (as opposed to `urandom`). See https://man7.org/linux/man-pages/man4/random.4.html

Hi @bergwolf , have you implemented this first? Any comment?
Cc @jodh-intel

Reply:
@wainersm - possibly, but more likely `getrandom(2)`. Certainly if there is insufficient entropy to boot the guest kernel we probably shouldn't continue anyway. However, all that is out of scope of this PR, which is primarily for a test change.